src/App.vue

```diff
@@ -272,7 +287,7 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     if (!this.haveChanges || force) {
       // redirect to dashboard
       const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
-      window.location.assign(dashboardUrl + this.nrNumber)
+      // TODO: window.location.assign(dashboardUrl + this.nrNumber)
```

Comment:
The only change here is to use the TRN instead of the NRN.
Eg, https://dev.bcregistry.ca/cooperatives/T123456789

Reply:
done